src/lib/components/Errors.svelte

```diff
@@ -13,7 +13,7 @@
 
 <ul>
 	{#each [...diagnostics] as diagnostic}
-		<li class={diagnostic.type} transition:slide={{ delay: 250, duration: 250 }}>
+		<li class={diagnostic.type} transition:slide|global={{ delay: 250, duration: 250 }}>
```

Comment:
Not sure we need this here but could also not matter

Reply:
It was added by the svelte migration tool...and I guess, why not 🤷‍♂️